src/main/java/seedu/contax/model/appointment/Appointment.java

```diff
@@ -120,7 +155,9 @@ public String toString() {
                 + "\n **Duration:** "
                 + getDuration()
                 + "\n **Person:** "
-                + (getPerson() == null ? "None" : getPerson().getName());
+                + (getPerson() == null ? "None" : getPerson().getName())
+                + "\n **Priority:** "
+                + getPriority();
```

Comment:
Can you add a check here that either leaves the line out or prints None for null priority?

Reply:
thx changed follow person